### ISSUE TITLE
(- 1 (first vals)) should be (sub1 (first vals)) or (- (first vals) 1)

### ISSUE THREAD

**Issue by wlz** [pkrumins/the-little-schemer#5]
In 10-value-of-all-of-this.ss , method apply-primitive , line 332 ,
primitive sub1 here should be the decrement operation ;
If use (- 1 (first vals)) , the value will be 1 - (first vals) in infix expression , however , the right answer should be (first vals) -1. ;-)

**Comment by Lupeipei** [comment]
@wlz You are right. primitive sub1 here is something wrong. the right answer should be `(- (first vals) 1)` .
